src/simulation/LoadGenerator.cpp

```diff
@@ -49,6 +50,10 @@ const uint32_t LoadGenerator::TX_SUBMIT_MAX_TRIES = 10;
 // ledger.
 const uint32_t LoadGenerator::TIMEOUT_NUM_LEDGERS = 20;
 
+// After successfully submitting desired load, wait for this many ledgers
+// without checking for account consistency.
+const uint32_t LoadGenerator::COMPLETION_TIMEOUT_WITHOUT_CHECKS = 4;
```

Comment:
Oh, I see. Should we just use one parameter across all loadgen modes for consistency? I think as soon loadgen is done submitting load, the transactions would either be picked up within the next 3 ledgers or dropped, so it _seems_ like there isn't much value in waiting for 20 ledgers (I can't recall how we landed on 20, it might have been an arbitrary choice. @MonsieurNicolas  might know better)

Reply:
I guess for the old constant we wouldn't normally reach the limit (unless something is broken), so there was no point in fine-tuning it. For the new constant we always have to wait this much. I'm not opposed to using a single small constant, though maybe would do that separately to be safe.